conf/evolutions/default/87.sql

```diff
@@ -0,0 +1,9 @@
+# --- !Ups
+
+ALTER TABLE `patients`
+ADD COLUMN `deleted_by_user_id` INT(11) NULL DEFAULT NULL  AFTER `isDeleted`;
```

Comment:
a foreign key constraint should be added between this column and the id column in the users table. This ensures the ID always exists in the users table